tf2onnx/schemas.py

```diff
@@ -156,7 +156,7 @@ def build_onnx_op(node):
     try:
         inferred_model = shape_inference.infer_shapes(model_proto)
     except Exception:  # pylint: disable=broad-except
-        logger.warning(
+        logger.info(
```

Comment:
I have liked that this is a warning since it often indicates a problem with freezing.

Reply:
done